### DISCUSSION TITLE
SyntaxError: multiple exception types must be parenthesized

### DISCUSSION THREAD

**Discussion by vikascod** [fastapi/fastapi#9683] [Questions] [answered]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

When I am importing some function from another file within same directory I am getting this error 

SyntaxError: multiple exception types must be parenthesized

I Have checked many times there is no any syntax error

![image](https://github.com/tiangolo/fastapi/assets/102589808/89b0d1dc-c398-4f45-9da7-1300b1e9524e)

**Comment by Kludex** [comment]
Use `except Except as e` instead of `except Except, e`.

**Comment by vikascod** [comment]
I am not using try and Except any where in my code

**Comment by Kludex** [comment, marked as answer]
It looks like it's from SQLAlchemy.
